backend/typescript/models/donor.model.ts

```diff
@@ -0,0 +1,25 @@
+import { Column, DataType, Model, Table, ForeignKey, BelongsTo } from "sequelize-typescript";
+import { DonorRole } from "../types"
+import User from "./user.model"
+
+@Table({ tableName: "donors" })
+export default class Donor extends Model {
+  @ForeignKey(() => User)
+  @Column({ type: DataType.INTEGER })
+  user_id!: number;
+
+  @BelongsTo(() => User)
+  user!: User;
+
+  @Column({ type: DataType.ENUM("LocalBusiness", "IndividualDonor") })
+  donor_type!: DonorRole;
+
+  @Column({ type: DataType.STRING })
+  facebook_link!: string;
+
+  @Column({ type: DataType.STRING })
+  instagram_link!: string;
+
+  @Column({ type: DataType.BOOLEAN })
+  recurring_donation!: boolean;
```

Comment:
Since these are optional fields, do we need to specify that somewhere or can we have null values right now like this?

Reply:
I think to make them optional we just change the ! to ? (Just pushed this change btw)

Reply:
https://www.npmjs.com/package/sequelize-typescript#table-api. Doesnt the documentation here has an allowNull attribute that we might need to add?